### PR TITLE
Chat topnav icons

### PR DESCRIPTION
**Pull request by falklinus**
# Jira Issue: []

## What?
- Use icons for assign, reject and resolve

## Why?
- Follow design

## Optional screenshots
image
image
image

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [x] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

File: src/portals/hope/features/tasks/list-items/TaskListItem.tsx [hunk 1]
```diff
@@ -98,25 +93,43 @@ const Name = styled.div<{ available: boolean }>`
...
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+`
 
-  color: ${({ theme }) => theme.accentContrast};
-  font-weight: bold;
-  font-size: 1rem;
+const show = keyframes`
```
Comment (vonElfvin) [inline src/portals/hope/features/tasks/list-items/TaskListItem.tsx:105]: Can we try an "opacity" swap and see how that works? Where one fades in and one fades out

File: src/portals/hope/features/tasks/list-items/TaskListItem.tsx [hunk 2]
```diff
@@ -208,55 +238,121 @@ export const TaskListItem: React.FC<
...
-              {title ? title : 'Not available'}
-            </Name>
-          </Flex>
-        </Spacing>
-        <span className="preview">{task.description}</span>
-        <Flex align="center">
+              {task.assignedTo ? convertEmailToSignature(task.assignedTo) : ''}
+            </UserCircle>
+          </Popover>
+          <MemberFlag>{flag}</MemberFlag>
```
Comment (vonElfvin) [inline src/portals/hope/features/tasks/list-items/TaskListItem.tsx:263]: Maybe we should swap to this? Where flag can be any of the Flags available in `constants.ts`
```suggestion
          <MemberFlag flag={flag} />
```

File: src/portals/hope/features/tasks/list-items/TaskListItem.tsx [hunk 3]
```diff
@@ -208,55 +238,121 @@ export const TaskListItem: React.FC<
...
+                    size="large"
+                    disabled={!assignedToMe}
+                    variant="tertiary"
+                    onClick={(e) => {
+                      e.stopPropagation()
+                      onReject()
+                    }}
+                    icon={<XCircle />}
+                  />
+                </Popover>
```
Comment (vonElfvin) [inline src/portals/hope/features/tasks/list-items/TaskListItem.tsx:329]: Any way we can extract these two very similar buttons into one Component that we use on the `TaskChat` as well? 🤔 One will have a hotkey and one won't. But it should be doable!

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 4]
```diff
@@ -452,44 +439,26 @@ const TasksPage: React.FC = () => {
                       </p>
                     </TopBarItem>
                   ))}
-                  <AssignedToDropdown
```
Comment (vonElfvin) [inline src/portals/hope/pages/tasks/TasksPage.tsx:455]: :poggers:

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 5]
```diff
@@ -452,44 +439,26 @@ const TasksPage: React.FC = () => {
...
-                      checkedIn ? 'You + ' : ''
-                    }${countCheckedInUsers} checked-in`}
+
+                  <Popup
+                    text={`Checked In (${checkedInUsers.length})`}
+                    style={{ marginRight: '2rem', marginLeft: 'auto' }}
                   >
                     {checkedInUsers
-                      .filter((user) => user.email !== email)
+                      // .filter((user) => user.email !== email)
```
Comment (vonElfvin) [inline src/portals/hope/pages/tasks/TasksPage.tsx:448]: Oupsie doopsie

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 6]
```diff
@@ -716,6 +718,7 @@ const TasksPage: React.FC = () => {
               slim={isMobile}
               assignedToMe={activeTask.assignedTo === email}
               task={activeTask}
+              userTaskCount={userTaskCount}
```
Comment (vonElfvin) [inline src/portals/hope/pages/tasks/TasksPage.tsx:721]: Seeing all of these props makes me think we should maybe move some stuff into a context 🤔 But maybe not now

**Review by vonElfvin — APPROVED** [review]
💯 💯

File: src/portals/hope/pages/tasks/TasksPage.tsx (hunk 6 above)
Comment (falklinus) [reply src/portals/hope/pages/tasks/TasksPage.tsx:721]: Sounds very nice 👍 Let's remember this

File: shared/@hedvig-ui/HotkeyHint/index.tsx [hunk 7]
```diff
@@ -164,7 +163,7 @@ export const Hint: React.FC<
       {...props}
     >
       {textPosition === 'left' && <p>{text}</p>}
-      {keys.map((key, index) => (
+      {keys?.map((key, index) => (
```
[the remark this answers is not in the document]
Comment (falklinus) [reply shared/@hedvig-ui/HotkeyHint/index.tsx:166]: Think it's exactly the same

**Pull request merged**